src/fsharp/creflect.fs

```diff
@@ -967,7 +967,7 @@ and ConvTyconRef cenv (tcref:TyconRef) m =
     | TProvidedTypeExtensionPoint info when not cenv.g.isInteractive && not info.IsErased -> 
         // Note, generated types are (currently) non-generic
         let tref = ExtensionTyping.GetILTypeRefOfProvidedType (info.ProvidedType, m)
-        ConvILTypeRef cenv tref
```

Comment:
@dsyme there is now only one use of `ConvILTypeRef`, below on line 978.  Can you take a second look and confirm that this call is correct?